docs/reference/modules/ROOT/pages/lucene.adoc

```diff
@@ -184,6 +181,45 @@ You can escape your input strings when constructing Lucene query strings by call
 
 This is helpful to mitigate against injection attacks and other errors.
 
+== Custom searching outside of Datalog
+
+The more direct `crux.lucene/search` function is available to lazily return results, without the temporal filtering or other constraints of using Lucene via the `q` API.
+
+The function accepts 3 parameters (`node`, `query` and `opts`) and returns an iterable cursor of results that must be closed.
+
+The `query` parameter can be either a Lucene query string or an `org.apache.lucene.search.Query` object.
+
+The `opts` parameter accepts a map with a single `:default-field` entry.
+The value of this entry will be supplied to the Lucene `QueryParser` in the cases where the supplied `query` parameter is a Lucene query string.
+
+[source,clojure]
+----
+(with-open [search-results (crux.lucene/search node "Ivan")]
+  (into [] (iterator-seq search-results)))
+----
+
+Each item returned will be a vector of `org.apache.lucene.document.Document` and a Double representing the matched score.
+
+See the https://github.com/juxt/crux/blob/master/crux-lucene/test/crux/lucene/extension_test.clj[extension tests] for examples of decoding the contents of the result document and performing userspace temporal filtering.
```

Comment:
I'm not generally a fan of pointing people at our tests within the docs, feels like it's putting the work onto the user to understand what's going on in our tests?

Reply:
Maybe something akin to the Corda reference docs would work better here?

Reply:
As discussed, one for a separate how-to